src/main/java/uk/gov/hmcts/ccd/endpoint/exceptions/DataProcessingException.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.ccd.endpoint.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.UNPROCESSABLE_ENTITY)
+public class DataProcessingException extends ValidationException {
+
+    public DataProcessingException() {
+        super("Processing of data failed");
```

Comment:
I think error reporting must provide more details about the cause of the failure. This message is too generic and won't help investigating the issue